sdk/src/main/kotlin/io/rover/sdk/ui/blocks/poll/image/ImagePollViewModel.kt

```diff
@@ -89,7 +92,7 @@ internal class ImagePollViewModel(
     override val multiImageUpdates = PublishSubject<Map<String, ImagePollViewModelInterface.ImageUpdate>>()
 
     private fun multiImageUpdate() {
-        measurementsSubject
+        measurementsSubject.first()
```

Comment:
perhaps .distinct() would be better here rather than .first(), because multiple emissions are definitely possible and the most recent one should be trusted.